docs/concepts.rst

```diff
@@ -228,6 +228,101 @@ Not like this, where the join task is skipped
 
 .. image:: img/branch_bad.png
 
+SubDAGs
+'''''''
+
+SubDAGs are perfect for repeating patterns. Defining a function that returns a
+DAG object is a nice design pattern when using Airflow.
+
+Airbnb uses the *stage-check-echange* pattern when loading data. Data is staged
```

Comment:
echange is French for exchange. Do not know if that is a nod to Airflow's French roots, but probably good to make it `exchange` for the anglophones.